### PR TITLE
Added stageSize, groundDensity, groundFrequency

### LINKED ISSUE #75: Change size of the environment
How to change the width or the size of the environment? I have tried using scaling, but that distorts the sky and other features.

### PR DESCRIPTION
**Pull request by PietroAmmaturo**
Resolves #75.
This should resolve the environment size issue. The user has now the possibility not only to enlarge the environment.
The user can also change the density (resolution) of the ground mesh and the frequency, particurarly useful when using the canyon ground, to obtain more interesting results.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,75 +8,75 @@ A simple way of setting up a whole basic environment for your
...
+| **groundColor**          | '#553e35'    | Main color of the ground                                                                                                                                                                                                                                                                     |
+| **groundColor2**         | '#694439'    | Secondary color of the ground. Used for textures, ignored if`groundTexture` is `none`                                                                                                                                                                                                        |
+| **groundDensity**        | '64'         | Number of divisions of the ground mesh.                                                                                                                                                                                                                                                      |
```
Comment (vincentfretin) [inline README.md]: It's a number, remove the quotes 64

File: README.md [hunk 2]
```diff
@@ -8,75 +8,75 @@ A simple way of setting up a whole basic environment for your
...
+| **groundFrequency**      | '10'         | Frequency used to generate noise / terrain unevenness on the ground mesh.                                                                                                                                                                                                                  |
```
Comment (vincentfretin) [inline README.md]: It's a number, remove the quotes 10

File: index.js [hunk 3]
```diff
@@ -28,12 +28,15 @@ AFRAME.registerComponent('environment', {
...
     playArea: {type: 'float', default: 1, min: 0.5, max: 10},
+    stageSize: {type: 'float', default: 200, min: 1, max: 20000},
 
     ground: {default: 'hills', oneOf:['none', 'flat', 'hills', 'canyon', 'spikes', 'noise']},
     groundYScale: {type: 'float', default: 3, min: 0, max: 50},
     groundTexture: {default: 'none', oneOf:['none', 'checkerboard', 'squares', 'walkernoise']},
     groundColor:  {type: 'color', default: '#553e35'},
     groundColor2: {type: 'color', default: '#694439'},
+    groundDensity: {type: 'number', default: 64, min: 8, max: 1024},
+    groundFrequency: {type: 'float', default: 10, min: 0.1, max: 1000},
```
Comment (vincentfretin) [inline index.js]: There is no float type, it's a number.

File: index.js [hunk 4]
```diff
@@ -571,7 +584,7 @@ AFRAME.registerComponent('environment', {
 
         // calculate next x,y ground coordinates
         x += inc;
-        if (x >= 10) {
+        if (x >= frequency ) {
```
Comment (vincentfretin) [inline index.js]: remove space before `)`

**Review by vincentfretin — COMMENTED** [review]
Great, allowing more configuration is always nice.

**Comment by PietroAmmaturo** [comment]
This is really complete, I am not sure hexactly about how to update the files: "dist\aframe-environment-component.js" and "dist\aframe-environment-component.min.js" to match my changes @vincentfretin (This is my first PR to an A-Frame component).
For dist\aframe-environment-component.js I could just copypasete, but for dist\aframe-environment-component.min.js?

**Comment by vincentfretin** [comment]
To build the dist, it's `npm install` and `npm run dist`, but don't include the files in the PR. @dmarcos will take care of creating the dist once the PR is merged.

**Comment by PietroAmmaturo** [comment]
Thank you, then it is ready to merge I belive.

**Comment by dmarcos** [comment]
Thanks so much!

**Comment by dmarcos** [comment]
I bump dist on master and also published a new version on npm. You all rock!

**Pull request merged**